.github/workflows/continuous-integration.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Check formatting with spotless.
       run: mvn spotless:check -B
     - name: Check formatting with Black.
-      run: black --fast --check --extend-exclude IDE --extend-exclude jython3 .
+      run: black --fast --check --extend-exclude IDE --extend-exclude jython3 --extend-exclude WALA .
```

Comment:
Somehow, `black` doesn't like more than two exclude extensions. Looking into this now.

Reply:
maybe we should run black explicitly on those projects we do want checked...